lib/game.rb

```diff
@@ -24,7 +24,6 @@ def self.all
   end
 
   def initialize(game_info)
-    @game_info = game_info
```

Comment:
we didn't need to save the game info, since we're parsing it into all the other data